mindocr/models/builder.py

```diff
@@ -75,4 +77,7 @@ def build_model(name_or_config: Union[str, dict], **kwargs):
     if "amp_level" in kwargs:
         auto_mixed_precision(network, amp_level=kwargs["amp_level"])
 
+    num_params = sum([param.size for param in network.get_parameters()])
+    _logger.info(f"Total number of model parameters: {num_params}")
```

Comment:
Would be nice to specify the number of trainable and non-trainable parameters as well.

Reply:
We may put this log info into the key info section  (=== ... ===) in train.py/eval.py for easy spotting.

Reply:
done

Reply:
since logs are separated now, set `num_trainable_params` for training only?

Reply:
I think evaluation script still needs this information, if user doesn't like to retrain but still need to spot this info